static/index.html

```diff
@@ -63,11 +63,44 @@
     </div>
   </div>
   <footer>
-    <div class="container">
-      <div class="col-md-12">
-        <div class="camp-logo-footer"><img src="images/drupalcebu_puso.svg" /></div>
-        <p>Since 2014, Drupalcamp Cebu is brought to you by the <a href="https://www.facebook.com/groups/drupalcebu/">Drupal Cebu User Group.</a> <a href="#">Terms of use.</a> <a href="#">Code of conduct.</a></p>
-        <p>Made with <i class="fa fa-heart" aria-hidden="true"></i> using Drupal 8. <a href="#">Access the source code.</a></p>
+    <div class="footer-top-wrapper">
+      <div class="container">
+        <div class="col-md-12">
+          <div class="block-footer-left col-md-4">
+            <h2>Drupal in Cebu</h2>
+            <p>This camp is brought to you by <a href="https://www.facebook.com/groups/drupalcebu/">Drupal Cebu User Group</a></p>
```

Comment:
Link to FB should open as a new tab. It's an external link (outside of our website).